instrumentation/resources/library/src/test/java/io/opentelemetry/instrumentation/resources/OsResourceTest.java

```diff
@@ -23,119 +24,119 @@ class OsResourceTest {
   @SetSystemProperty(key = "os.name", value = "Linux 4.11")
   void linux() {
     Resource resource = OsResource.buildResource();
-    assertThat(resource.getSchemaUrl()).isEqualTo(ResourceAttributes.SCHEMA_URL);
-    assertThat(resource.getAttribute(ResourceAttributes.OS_TYPE))
-        .isEqualTo(ResourceAttributes.OsTypeValues.LINUX);
-    assertThat(resource.getAttribute(ResourceAttributes.OS_DESCRIPTION)).isNotEmpty();
+    assertThat(resource.getSchemaUrl()).isEqualTo(SchemaUrls.V1_24_0);
```

Comment:
[for reviewer] I am not sure what is the "right" value to pick here, so I picked the 1.24.0 to match the semconv version.